<launch>
    <!-- declare arg to be passed in -->
    <arg name="is_show" /> 
    <node name="detect_track_node" pkg="detect_track" type="detect_track_node" args="$(arg is_show)" output="screen">
        <param name="h_low_greenBG" value="40"/>
        <param name="h_high_greenBG" value="80"/>
    </node>
</launch>
